contrib/docker_compose_workers/README.md

```diff
@@ -94,20 +94,6 @@ worker_replication_host: synapse
 worker_replication_http_port: 9093
 ```
 
-### Add Workers to `instance_map`
-
-Locate the `instance_map` section of your `homeserver.yaml` and populate it with your workers:
-
-```yaml
-instance_map:
-  synapse-generic-worker-1:        # The worker_name setting in your worker configuration file
-    host: synapse-generic-worker-1 # The name of the worker service in your Docker Compose file
-    port: 8034                     # The port assigned to the replication listener in your worker config file
```

Comment:
An `instance_map` is only is required for stream writers.
See also:
https://github.com/matrix-org/synapse/blob/1bf2832714abdfc5e10395e8e76aecc591ad265f/docker/configure_workers_and_start.py#L275-L286

Stream writers are not part of this example and explanation and need some more configuration eg `stream_writers`.